README.md

```diff
@@ -26,11 +26,11 @@ steps:
       restore_cache $(hash_file package-lock.json)
 
     plugins:
-      - automattic/a8c-ci-toolkit#3.0.0:
+      - automattic/a8c-ci-toolkit#3.0.1:
```

Comment:
The next version that we'll do and that will include this change will likely be a `3.1.0`, since this new command  is a New Feature, not a bugfix.

```suggestion
      - automattic/a8c-ci-toolkit#3.1.0:
```

Reply:
I was actually fixing `trunk`, as it still specifies `3.0.0` and I was getting a [lint error](https://buildkite.com/automattic/ci-toolkit-buildkite-plugin/builds/564#018dccf0-9182-4d38-aa5f-f7ba2db690fc/125-141) due to that.

I wasn't sure if this PR should already do the preparation for the 3.1.0 release (I agree it should be there) or if I should leave it to another PR (as I saw on https://github.com/Automattic/a8c-ci-toolkit-buildkite-plugin/pull/77).

Reply:
Ah, gotcha 👍 

Yeah, I think it makes more sense to fix it in the (future) release preparation PR indeed.